spacy/errors.py

```diff
@@ -542,6 +542,8 @@ class Errors(metaclass=ErrorsWithCodes):
     E202 = ("Unsupported {name} mode '{mode}'. Supported modes: {modes}.")
 
     # New errors added in v3.x
+    E852 = ("The tar file pulled from the remote attempted an unsafe path "
+            "traversal.")
```

Comment:
Not important to adding the PR, but why is the error number lower than existing ones rather than higher?